chart/README.md

```diff
@@ -20,17 +20,18 @@ recommend deploying your main NATS server with another chart.
 
 ## Running the chart
 
-The chart is available in a hosted Helm repository. To add the repository, run the following:
+You can install the chart via [the `wasmcloud-chart` package on ArtifactHub][artifacthub-wasmcloud]:
 
 ```console
-$ helm repo add wasmcloud https://wasmcloud.github.io/wasmcloud-otp/
+$ helm install wasmcloud oci://ghcr.io/wasmcloud/wasmcloud-chart --version 0.7.0
```

Comment:
Will we have to be updating this every time we bump wasmCloud version? That seems like an awful lot of maintenance burden

Reply:
Not necessarily, the Chart should be independently versioned based on changes to the underlying software it deploys that necessitate changes to the Chart.

For example, if we choose to add or remove configuration options, we should update the Chart to reflect that.

However, new versions of wasmCloud being released where nothing else changes, it would be fine (imo) to keep the Chart as is, because the Chart provides a way to override the version of wasmCloud host it's running via `image.tag` option (and falling back to the Chart's appVersion if not provided): https://github.com/wasmCloud/wasmCloud/blob/main/chart/templates/deployment.yaml#L35